<launch>
  <!-- node parameters -->
  <arg name="map_topic" default="/realtime_cost_map" />
  <arg name="max_distance" default="3.0" />

  <!-- Launch the grid map simple demo node -->
  <node pkg="object_map" type="dist_transform" name="dist_transform" output="log">
    <param name="map_topic" value="$(arg map_topic)" />
    <param name="max_distance" value="$(arg max_distance)" />
  </node>

<!--
  <node pkg="tf" type="static_transform_publisher" name="occupancy_grid_transform" args="0 0 -2.0 0 0 0 velodyne occupancy_grid 10" />
-->

  <!-- Launch the grid map visualizer -->
  <node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="log">
    <rosparam command="load" file="$(find object_map)/config/dist_transform.yaml" />
  </node>
</launch>
